README.md

```diff
@@ -94,6 +96,20 @@ to run e2e martin tests against the application:
 make up integration
 ```
 
+### Over-the-air (OTA) Update
+
+To deploy your edits to the phylum on to the running fabric network, simply run:
```

Comment:
Is it clear enough to just say "on to the running fabric network"? I guess it's described directly above but if this is a new section someone might skip directly here. They also don't know about `make down` yet. Would it be good to tell them about `make down` first and then have this section tell them to run `make up` again?